repoload/repoload.py

```diff
@@ -151,13 +168,84 @@ def get_changes_for_topic(topic):
     return changes
 
 
-def get_change(number):
-    changes, _ = query(URL, "%s" % (number,))
+def get_change(number, options=None):
+    changes, _ = query(URL, "%s" % (number,), options=options)
     if len(changes) != 1:
         raise Exception("Cannot find change for id %s" % (number,))
     return changes[0]
 
 
+def show_change(args):
+    # Show information about a change id
+    number = args.value
+    change = get_change(number, options=[GerritOptions.CommitMessage,
+                                         GerritOptions.CurrentPatchSet,
+                                         GerritOptions.PatchSets,
+                                         GerritOptions.Comments])
+    print_change(change)
+
+
+def print_change(change):
+    w = TextWrapper(width=72, initial_indent="\t", subsequent_indent="\t")
+    c = change
+    print("Project:\t{} @ {} ".format(c["project"], c["branch"]))
+    print("Status:\t\t{}".format(c["status"]))
+    if c["open"] == "true":
+        print("Cr is open")
+
+    print("Change id:\t{}".format(c["id"]))
+    print("Change number:\t{}".format(c["number"]))
+
+    print("Created at\t{}".format(datetime.fromtimestamp(c["createdOn"])))
+    print("Last updated\t{}".format(datetime.fromtimestamp(c["lastUpdated"])))
+    print("Url\t\t{}".format(c["url"]))
+
+    print("Change subject:\t{}".format(c["subject"]))
+    print("Change owner:\t{} ({})".format(c["owner"]["name"], c["owner"]["email"]))
+
+    print('\nCommit Message:\n')
+    for line in str.splitlines(c["commitMessage"]):
+        print(w.fill(line))
```

Comment:
At least for the commit message I don't think it's useful to line wrapping it in the repolint tool. The commit message is normally already correctly line wrapped. Only long lines like URLs, compile output or log message are sometimes longer than 72 characters.

I think it's better to not modify the commit message to have a consistent and easier to understand behavior of repolint.